tox.ini

```diff
@@ -41,6 +41,6 @@ whitelist_externals=black
 commands=
     mypy -p <MODULE_NAME> --config-file {toxinidir}/mypy.ini
     flake8 {toxinidir}/<MODULE_NAME> {toxinidir}/tests
-    isort --check-only --diff {toxinidir}/<MODULE_NAME> {toxinidir}/tests
+    isort --check-only --diff --multi-line=VERTICAL_HANGING_INDENT --force-grid-wrap=1 --combine-as {toxinidir}/<MODULE_NAME> {toxinidir}/tests
```

Comment:
nit: I saw some chatter about `--combine-as` switching the default from `false` to `true`. Enough so that I'm not sure what it's settled on but I think it will only default to `true` in isort 6+. That being said, and regardless of what it lands on, I'd rather have an explicit value for the flag here.

Reply:
Turns out we already have `combine_as_imports=True` earlier in the file, so I just kept that. Also moved these CLI options up so we have all config in one place.